config/legacy_static_images.json

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
looks like the legacy image json will be generated in the script...do we still need this file?

Reply:
We are keeping this for now because the way the code works is that the image before a major or minor version bump will become the new legacy. The first time this script runs there will be no major or minor version bump so the static images are the legacy. This can be removed once there is a major or minor version bump for both the Bioconductor and GATK images.